telemetry/definitions/commonDefinitions.json

```diff
@@ -3294,6 +3438,419 @@
                 { "type": "codecatalyst_devEnvironmentWorkflowStep"},
                 { "type": "codecatalyst_devEnvironmentWorkflowError", "required": false }
             ]
+        },
+        {
+            "name": "codeTransform_isVisibleToUser",
+            "passive": true,
```

Comment:
This one is weird because we implemented before where the transform button was and was NOT visible to users. If the users had an invalid project, we opted NOT to show the button.

We changed that so I'm not sure we need this metric anymore. Looking at removing.

Reply:
Removing. Its a NOP.